plenum/server/node.py

```diff
@@ -132,6 +133,8 @@ def __init__(self,
         self.basedirpath = basedirpath or config.baseDir
         self.dataDir = self.config.nodeDataDir or "data/nodes"
 
+        self._view_change_timeout = self.config.VIEW_CHANGE_TIMEOUT
```

Comment:
We need to have a timeout for election as well (when we elect a first primary in round-robin fashion, there may be no view change; since this is round-robin - we need to have a timeout to make sure that the new primary is working)